roles/hammerdb/templates/db_workload.yml.j2

```diff
@@ -17,9 +17,12 @@ spec:
         imagePullPolicy: Always
         wait: true
         command: ["/bin/sh", "-c"]
-        args: 
-          - "/usr/local/bin/uid_entrypoint; 
+        args:
+          - "/usr/local/bin/uid_entrypoint;
              {% if elasticsearch is defined %}
+             {% if elasticsearch.cert_verify is defined %}
+             export es_verify_cert={{ elasticsearch.cert_verify }};
+             {% endif %}
              {% if elasticsearch.server is defined %}
              export es={{elasticsearch.server}};
              export es_port={{elasticsearch.port}};
```

Comment:
Why don't we set these environment variables in the containers.env section?. This will make code clearer.
I think the same for the remaining exported variables

Reply:
I think this is a fine suggestion. This is mostly historical, *why* we export vs the pod def.